### DISCUSSION TITLE
How do I Create Multiple Related Records in One Form?

### DISCUSSION THREAD

**Discussion by AlexandriaLittle** [Budibase/budibase#5665] [Help]
Hi All,

I'm trying to replace a few paper processes at my workplace. One of them is a form that requests a check be written for a specific person, use, etc. and the money for the check can come from one or more budget accounts to fund that check. I'm really struggling on making the which-account(s)-to-fund-from piece. Ideally, I'd like the form to be able to be broken into steps where the first section asks general questions about the request and the second section allows the user to specify as many budget accounts as they need to draw from. I want the users to be able to specify custom budget accounts so I can't use multi-select either. :/

My BudibaseDB schema is as follows:
One Check Request -> One Check
One Check -> Many Budget Accounts

How could I accomplish this?

**Comment by melohagan** [comment]
Hi @AlexNCurrie 

You could try having a look at these:

- Form Steps: https://docs.budibase.com/docs/form-steps
- Linking Table (similar to your setup): https://github.com/Budibase/budibase/discussions/5299#discussioncomment-2530261
